src/main/java/example/Settings.java

```diff
@@ -65,28 +66,26 @@ public int getPort() {
     }
 
     @Override
-    public Map<String, Swagger> enableSwagger() {
-        EntityDictionary dictionary = new EntityDictionary(new HashMap());
-
-        dictionary.bindEntity(ArtifactGroup.class);
-        dictionary.bindEntity(ArtifactProduct.class);
-        dictionary.bindEntity(ArtifactVersion.class);
-        Info info = new Info().title("Test Service").version("1.0");
-
-        SwaggerBuilder builder = new SwaggerBuilder(dictionary, info);
-
-        Swagger swagger = builder.build().basePath("/api/v1");
-
-        Map<String, Swagger> docs = new HashMap<>();
-        docs.put("test", swagger);
-        return docs;
+    public boolean enableSwagger() {
+        return true;
     }
 
     @Override
     public String getModelPackageName() {
-
         //This needs to be changed to the package where your models live.
         return "example.models";
+//    	return "dynamicconfig.models";
+    }
+    
+    @Override
+    public boolean enableDynamicModelConfig() {
+        return true;
+    }
+    
+    @Override
+    public String getDynamicConfigPath() {
+        File file = new File("src/main/resources/models");
```

Comment:
do we need to use File and absolute path? in the standalone code unit test  we just return `src/test/resources/models`